packages/dialtone-css/.github/CONTRIBUTING.md

```diff
@@ -87,7 +86,7 @@ Note that we use lesshint only for the utilities folder and stylelint for everyt
 Here are some important directories to know within the Dialtone repository
 
 - `lib/build`: All source code and assets for the Dialtone library.
-- `lib/dist`: The compiled bundle will be output here upon `nx build dialtone-css`.
+- `lib/dist`: The compiled bundle will be output here upon `nx run dialtone-css:build`.
```

Comment:
> Should it be `pnpm nx` in case they don't have nx installed locally? I saw you used that in some other examples

Used `pnpm nx` only on `tasks.json` which is for VSCode, did that because I had problems previously when VSCode didn't found nx so commands didn't run properly, hopefully that fixes that issue. But for the rest, I'll prefer to run `nx run ...`